src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js

```diff
@@ -61,6 +62,7 @@ export function getReasonCode({ data, isCC, isAcheron, isDS }) {
         : null,
   };
   if (isAcheron) return reasonCodeBody;
+  // the below return is utilized when Acheron is off - need to cleanup
   return {
```

Comment:
Is this comment removal intended as well ?

Reply:
The comment is added as reminder when we do cleanup for Acheron toggle flag removal.